src/embed.js

```diff
@@ -24,6 +26,26 @@ matchers.push(unsplash);
 matchers.push(lottie);
 matchers.push(spotify);
 
+/**
+ * function reads file and returns json object
+ * @param {string} file name of file to be loaded
+ * @param {object} log handle on logging object
+ * @returns json object of fastly public ips
+ */
+async function getIPList(file, log = console) {
+  const fd = path.resolve(file);
```

Comment:
why `fd` ?

Reply:
And wouldn't a plain `require` work, too?